subworkflows/nf-core/fastq_align_parabricks/tests/nextflow.config

```
@@ -0,0 +1,10 @@
+process {
+
+    withName: 'PARABRICKS_FQ2BAM' {
+        ext.args = '--low-memory'
+    }
+    // Ref: https://forums.developer.nvidia.com/t/problem-with-gpu/256825/6
+    // Parabricks’s fq2bam requires 24GB of memory.
```

Should we maybe do a 
```suggestion
    // Parabricks’s fq2bam requires 24GB of memory.
    // resourceLimits = [cpus: 6GB, memory: 24.GB]
    memory = '24.GB'
```

https://docs.nvidia.com/clara/parabricks/latest/documentation/tooldocs/man_fq2bam.html#man-fq2bam

Also it would be awesome to do some thing like `--memory-limit ${task.memory} / 2` by default or make sure there's 16 cpus per GPU requested.

Just trying to push the resourceLimits syntax to the limits here 😆